plugins/node/opentelemetry-instrumentation-express/src/types.ts

```diff
@@ -88,6 +88,20 @@ export type SpanNameHook = (
   defaultName: string
 ) => string;
 
+/**
+ * Function that can be used to add custom attributes to the current span or the root span on
+ * a Express request
+ * @param requestSpan - The Express request handler span
+ * @param expressRequest: - The Express request object
+ */
+export interface ExpressRequestCustomAttributeFunction {
+  (
+    span: Span,
+    layerType: ExpressLayerType,
+    expressRequest: PatchedRequest
```

Comment:
Any reason to use `PatchedRequest` instead of `Request`?

Reply:
Now it uses an `ExpressRequestInfo` as suggested